polkaj-scale/src/test/groovy/io/emeraldpay/polkaj/scale/ScaleCodecWriterSpec.groovy

```diff
@@ -68,6 +68,14 @@ class ScaleCodecWriterSpec extends Specification {
         Hex.encodeHexString(act) == "ffffff00"
     }
 
+    def "Write unsigned 64-bit long"() {
+        when:
+        codec.writeUint64(2147483647L)
```

Comment:
This is within the range of a Long, using this 18446744073709551615 you'll never get there